pylint/pyreverse/printer.py

```diff
@@ -25,6 +25,7 @@ class EdgeType(Enum):
     ASSOCIATION = "association"
     AGGREGATION = "aggregation"
     USES = "uses"
+    TYPE_USES = "type_uses"
```

Comment:
```suggestion
    TYPE_DEPENDENCY = "type_dependency"
```
I think this gets the point a little bit better across, and is more consistent with the new `type_depends` attribute.